tests/Fixtures/Integration/priority/declare_strict_types,single_blank_line_before_namespace.test

```diff
@@ -5,7 +5,8 @@ Integration of fixers: declare_strict_types,single_blank_line_before_namespace.
 --REQUIREMENTS--
 {"php": 70000}
 --EXPECT--
-<?php declare(strict_types=1);
+<?php
+declare(strict_types=1);
```

Comment:
Matches (proposed) PSR-12 recommendation.
https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#25-keywords-and-types

>  [...]If present, each of the blocks below MUST be separated by a single blank line, and MUST NOT contain a blank line.[...]
> * [...]
> * One or more declare statements.